tests/components/history/test_init.py

```diff
@@ -252,7 +258,12 @@ def test_get_significant_states_entity_id(self):
         del states["script.can_cancel_this_one"]
 
         hist = history.get_significant_states(
-            self.hass, zero, four, ["media_player.test"], filters=history.Filters()
+            self.hass,
+            zero,
+            four,
+            ["media_player.test"],
+            filters=history.Filters(),
+            significant_changes_only=True,
```

Comment:
These tests should not be updated as it indicates a breaking change to the API.

Reply:
Done